test/unit/views/StartViewSpec.scala

```diff
@@ -76,9 +77,9 @@ class StartViewSpec extends UnitViewSpec {
       view.getElementById("contents").text() mustBe "startPage.contents.header"
 
       view.getElementById("contents-list") must haveChildCount(3)
-      view.getElementById("contents-list").child(0).text() mustBe "startPage.beforeYouStart.header"
-      view.getElementById("contents-list").child(1).text() mustBe "startPage.informationYouNeed.header"
-      view.getElementById("contents-list").child(2).text() mustBe "startPage.reportYourArrivalAndDeparture.header"
+      view.getElementById("contents-list").child(0).text() must include("startPage.beforeYouStart.header")
+      view.getElementById("contents-list").child(1).text() must include("startPage.informationYouNeed.header")
+      view.getElementById("contents-list").child(2).text() must include("startPage.reportYourArrivalAndDeparture.header")
```

Comment:
`mustBe` is better than `must include`. Are we able to check the correct message using `mustBe`?

Reply:
Done :)